### PR TITLE
Add BTC price formatting

### PR DESCRIPTION
**Pull request by easyuxd**
Add comma to BTC/Fiat Price

Simple change using f-strings. From a total programming noob, feel free to edit.

### CODE REVIEW

**Comment by kdmukai** [comment]
Have to implement this as a separate function, otherwise you break other calls to `getPrice()`:
https://github.com/marc3linho/OrangeClock/blob/1e6277d36bf6e834549b5663cbece4de7fa1c3c9/src/orangeClockFunctions/displayBlockMoscowFees.py#L70

**Comment by easyuxd** [comment]
@kdmukai 
Thanks. Added a separate getFiatPrice function, per your recommendation.

It's quite clunky. I'm sure there's a more elegant way to implement this -- feel free to propose an alternate solution if so.

File: src/orangeClockFunctions/displayBlockMoscowFees.py [hunk 1]
```diff
@@ -70,6 +70,14 @@ def getMoscowTime():
     return moscowTime
 
 
+def getFiatPrice(currency):
```
Comment (kdmukai) [inline src/orangeClockFunctions/displayBlockMoscowFees.py:73]: A more typical name would be something like "getPriceDisplay" (I think this is my preferred option) or "getPriceStr" to make it clear that it's related to "getPrice" but it returns a formatted string.

File: src/orangeClockFunctions/displayBlockMoscowFees.py [hunk 2]
```diff
@@ -70,6 +70,14 @@ def getMoscowTime():
     return moscowTime
 
 
+def getFiatPrice(currency):
+    gc.collect()
+    data = urequests.get("https://mempool.space/api/v1/prices")
+    price = data.json()[currency]
+    fiatPrice = f"{price:,}"
```
Comment (kdmukai) [inline src/orangeClockFunctions/displayBlockMoscowFees.py:77]: This repetition of code from "getPrice" violates DRY (Don't Repeat Yourself).

Simpler to just do:
```python
def getPriceDisplay(currency):
    return f"{getPrice(currency):,}"
```

Could also add a nice-to-have nod to the Europeans (I think they all swap comma and period separators):
```python
    ...
    price_str = f"{getPrice(currency):,}"
    if currency == "EUR":
        price_str = price_str.replace(",", ".")
    return price_str
```

File: src/orangeClockFunctions/displayBlockMoscowFees.py [hunk 3]
```diff
@@ -185,10 +193,10 @@ def main():
                 textRow2 = getMoscowTime()
             elif dispVersion2 == "fp1":
                 symbolRow2 = "K"
-                textRow2 = str(getPrice("USD"))
+                textRow2 = str(getFiatPrice("USD"))
```
Comment (kdmukai) [inline src/orangeClockFunctions/displayBlockMoscowFees.py:196]: Don't need the cast to string (`str()`) now that the function is returning a string.

**Comment by easyuxd** [comment]
@kdmukai Beautiful, clean code! Thanks for walking me through this -- it's a helpful learning process.

And replacing the comma for EUR is a nice detail. It was bugging me as well.

Re-compiled gui/fonts/libreFranklinBold56.py to include "."

File: src/orangeClockFunctions/displayBlockMoscowFees.py (hunk 2 above)
Comment (marc3linho) [reply src/orangeClockFunctions/displayBlockMoscowFees.py]: Great feature to swap , with . for the EUR

**Comment by marc3linho** [comment]
great idea and implementation @easyuxd and thanks for the review @kdmukai

**Review by marc3linho — APPROVED** [review]

**Pull request merged**